src/BlockChain/custom/New-AzBlockchainMember.ps1

```diff
@@ -0,0 +1,200 @@
+  [StartIPAddress <String>]: Gets or sets the start IP address of the firewall rule range.
+.Link
+https://docs.microsoft.com/en-us/powershell/module/az.blockchain/new-azblockchainmember
+#>
+function New-AzBlockchainMember {
+[OutputType([Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Models.Api20180601Preview.IBlockchainMember])]
+[CmdletBinding(DefaultParameterSetName='CreateExpanded', PositionalBinding=$false, SupportsShouldProcess, ConfirmImpact='Medium')]
+[Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Profile('latest-2019-04-30')]
+param(
+    [Parameter(Mandatory)]
+    [Alias('BlockchainMemberName')]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Path')]
+    [System.String]
+    # Blockchain member name.
+    ${Name},
+
+    [Parameter(Mandatory)]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Path')]
+    [System.String]
+    # The name of the resource group that contains the resource.
+    # You can obtain this value from the Azure Resource Manager API or the portal.
+    ${ResourceGroupName},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Path')]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Runtime.DefaultInfo(Script='(Get-AzContext).Subscription.Id')]
+    [System.String]
+    # Gets the subscription Id which uniquely identifies the Microsoft Azure subscription.
+    # The subscription ID is part of the URI for every service call.
+    ${SubscriptionId},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.String]
+    # Gets or sets the consortium for the blockchain member.
+    ${Consortium},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.Security.SecureString]
+    # Sets the managed consortium management account password.
+    ${ConsortiumManagementAccountPassword},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.String]
+    # Gets the display name of the member in the consortium.
+    ${ConsortiumMemberDisplayName},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.String]
+    # Gets the role of the member in the consortium.
+    ${ConsortiumRole},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Models.Api20180601Preview.IFirewallRule[]]
+    # Gets or sets firewall rules
+    # To construct, see NOTES section for FIREWALLRULE properties and create a hash table.
+    ${FirewallRule},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.String]
+    # The GEO location of the blockchain service.
+    ${Location},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Blockchain.Category('Body')]
+    [System.Security.SecureString]
+    # Sets the basic auth password of the blockchain member.
```

Comment:
Should have ValidateNotNullOrEmpty attribute, or the code below should account for case where parameter is bound and null or empty.